data/test/scenarios/wml_tests/UnitsWML/AbilitiesWML/special_damage_type.cfg

```diff
@@ -0,0 +1,314 @@
+#####
+# API(s) being tested: [damage]replacement_type.
```

Comment:
No period either - this isn't a sentence.

Reply:
i don't understand